docs/docs/functions/macros.md

```diff
@@ -39,11 +39,34 @@ Defines new blocks that can be inserted into an existing chain.
 
 Defines a new chain that isn't looped.
 
+Such a chain executes only once when invoked (via `run <node>`).
```

Comment:
I think we should take the habit of using parentheses since that's the syntax it will end up being in the code.
```suggestion
Such a chain executes only once when invoked (via `(run <node>)`).
```

Reply:
Fixed